.teamcity/src/subprojects/release/publishing/ProjectPublishing.kt

```diff
@@ -0,0 +1,28 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import subprojects.*
+import subprojects.build.core.*
+
+data class Build(val name: String, val build: BuildType?)
```

Comment:
If it's impossible to have `Build` without BuildType then it shouldn't be nullable

Reply:
Technically it should be impossible, but cannot be guaranteed at this point.

Reply:
So you have some builds without build types right now?